### PR TITLE
[TTAHUB-1874] Change permission name from "COLLABORATOR_TRAINING_REPORTS" to "POC_TRAINING_REPORTS"

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

- Change permission name in DB, backend, and UI. 
- Also remove feature flag and instead render URLs and views conditionally

## How to test

Confirm permissions function as they did before change (another change will be incoming to align with the current expectation).

Confirm only those with one of the three appropriate permissions (or admins) see the "Training Reports" menu link

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1874

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by kryswisnaskas** [comment]
I tried to connect as another user (Adam in this case) to enter myself as a POC, but with this Adam was able to see all of the pages except for the "Training reports" one. I am including the screenshot below. I was also unable to impersonate him. It didn't matter whether the flag was on or off.

image

**Comment by thewatermethod** [comment]
Thanks for finding that, @kryswisnaskas. I think it's ready for another look.

**Comment by thewatermethod** [comment]
Per conversation at standup today (7/24) I reverted the changes to the feature flag

File: frontend/src/components/SiteNav.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ const SiteNav = ({
                       Activity Reports
                     </NavLink>
                   </li>
-                  <FeatureFlag flag="training_reports">
+                  <FeatureFlag flag="training_reports" renderNotFound>
```
Comment (AdamAdHocTeam) [inline frontend/src/components/SiteNav.js:92]: Do we want a not found render in the links?

image

**Review by AdamAdHocTeam — APPROVED** [review]
Overall looks good aside from the one comment.

**Pull request merged**